Classes/Plugin/ListView.php

```diff
@@ -163,7 +163,8 @@ protected function getEntry($number, $template)
                         }
                         $conf = [
                             'useCacheHash' => 1,
-                            'parameter' => $this->conf['targetPid'],
+                            'parameter' => $this->conf['linkToPurl'] ? ($this->list[$number]['purl'] . '/' . $this->list[$number]['page']) : $this->conf['targetPid'],
```

Comment:
This is a very SLUB-specific way of creating the PURL as it assumes that the resolver knows how to handle PURL with concatenated page numbers... 😞